Makefile

```diff
@@ -0,0 +1,23 @@
+VERSION = $(shell git describe --tags --dirty)
+
+.PHONY: zfs-cleaner
+
+all: zfs-cleaner
+
+deb: zfs-cleaner
+	dpkg-buildpackage -uc -us
+
+test:
+	go test ./... -cover
```

Comment:
might as well give it the `-race` flag ?

Reply:
zfs-cleaner doesn't use any goroutines by itself, but I guess it won't hurt (anything but test performance). Added in ec0aeb3.